recipes/sdl2_mixer/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Make sure TravisCI can find SDL2
+if [ `uname` == Darwin ]; then
+	export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+fi
+
+sed -i -- "s|@prefix@|${PREFIX}|g" SDL2_mixer.pc.in
+SMPEG_CONFIG="${PREFIX}/bin/smpeg2-config"
+./configure --disable-dependency-tracking --prefix=${PREFIX}
```

Comment:
Ok, thanks for trying. Not everything has an easy way to run tests. Probably open an issue in the feedstock on this point.